fix(services)_: fix imported and not used error

jakubgs opened the pull request:
Fixes:
```
services/ext/service.go:7:2: "errors" imported and not used
```
And fixes errors like this:
```
messaging/core.go:421:3: cannot use params.timeSource
(variable of type "github.com/status-im/status-go/internal/timesource".TimeSource)
as *"github.com/status-im/status-go/timesource".NTPTimeSource value in argument to wakuv2.New: need type assertion
```
Issues introduced in:
- https://github.com/status-im/status-go/pull/6829

jakubgs committed:
fix(services)_: fix imported and not used error

Fixes:
services/ext/service.go:7:2: "errors" imported and not used

Introduced in:
https://github.com/status-im/status-go/pull/6829

Signed-off-by: Jakub Sokołowski <jakub@status.im>

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (35)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 564b8446 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/android/job/PR-6836/1/) | 2025-08-14 10:36:31 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250814-103353-564b84-pr6836.aar) |
| :heavy_check_mark: | 564b8446 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6836/1/) | 2025-08-14 10:37:09 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-103400-564b84-pr6836.zip) |
| :heavy_check_mark: | 564b8446 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6836/1/) | 2025-08-14 10:37:29 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250814-103354-564b84-pr6836.zip) |
| :heavy_check_mark: | 564b8446 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6836/1/) | 2025-08-14 10:38:42 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-103403-564b84-pr6836.zip) |
| :heavy_check_mark: | 564b8446 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/ios/job/PR-6836/1/) | 2025-08-14 10:39:36 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250814-103359-564b84-pr6836.zip) |
| :heavy_check_mark: | 564b8446 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6836/1/) | 2025-08-14 10:40:32 | ~6 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250814-103409-564b84-pr6836.zip) |
| :heavy_check_mark: | 564b8446 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/1/) | 2025-08-14 10:41:57 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/1//consoleText) |
| :x: | 564b8446 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/1/) | 2025-08-14 10:43:34 | ~9 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/1/consoleText) |
| :heavy_check_mark: | 564b8446 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/1/) | 2025-08-14 10:57:55 | ~23 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/1//consoleText) |
| :x: | 564b8446 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/2/) | 2025-08-14 11:00:24 | ~6 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/2/consoleText) |
| | | | | | | |
| :heavy_check_mark: | 29fe315b | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/android/job/PR-6836/2/) | 2025-08-14 11:20:26 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250814-111744-29fe31-pr6836.aar) |
| :heavy_check_mark: | 29fe315b | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6836/2/) | 2025-08-14 11:20:56 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250814-111745-29fe31-pr6836.zip) |
| :heavy_check_mark: | 29fe315b | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6836/2/) | 2025-08-14 11:21:01 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-111751-29fe31-pr6836.zip) |
| :heavy_check_mark: | 29fe315b | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6836/2/) | 2025-08-14 11:23:04 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-111757-29fe31-pr6836.zip) |
| :heavy_check_mark: | 29fe315b | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/ios/job/PR-6836/2/) | 2025-08-14 11:23:40 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250814-111749-29fe31-pr6836.zip) |
| :heavy_check_mark: | 29fe315b | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/2/) | 2025-08-14 11:25:26 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/2//consoleText) |
| :heavy_check_mark: | 29fe315b | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6836/2/) | 2025-08-14 11:26:40 | ~8 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250814-111802-29fe31-pr6836.zip) |
| :x: | 29fe315b | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/3/) | 2025-08-14 11:27:50 | ~10 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/3/consoleText) |
| :heavy_check_mark: | 29fe315b | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/2/) | 2025-08-14 11:42:59 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 24d74557 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/android/job/PR-6836/3/) | 2025-08-14 11:23:23 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250814-112032-24d745-pr6836.aar) |
| :heavy_check_mark: | 24d74557 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6836/3/) | 2025-08-14 11:24:11 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250814-112104-24d745-pr6836.zip) |
| :heavy_check_mark: | 24d74557 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6836/3/) | 2025-08-14 11:25:02 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-112106-24d745-pr6836.zip) |
| :x: | 24d74557 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/4/) | 2025-08-14 11:26:33 | ~7 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/4/consoleText) |
| :heavy_check_mark: | 24d74557 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6836/3/) | 2025-08-14 11:27:07 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-112309-24d745-pr6836.zip) |
| :heavy_check_mark: | 24d74557 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/ios/job/PR-6836/3/) | 2025-08-14 11:30:02 | ~6 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250814-112349-24d745-pr6836.zip) |
| :heavy_check_mark: | 24d74557 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/3/) | 2025-08-14 11:32:44 | ~7 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/3//consoleText) |
| :heavy_check_mark: | 24d74557 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6836/3/) | 2025-08-14 11:33:51 | ~7 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250814-112643-24d745-pr6836.zip) |
| | | | | | | |
| :heavy_check_mark: | 25f06d57 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/android/job/PR-6836/4/) | 2025-08-14 11:33:02 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250814-113044-25f06d-pr6836.aar) |
| :heavy_check_mark: | 25f06d57 | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6836/4/) | 2025-08-14 11:33:38 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250814-113046-25f06d-pr6836.zip) |
| :heavy_check_mark: | 25f06d57 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6836/4/) | 2025-08-14 11:33:58 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-113054-25f06d-pr6836.zip) |
| :heavy_check_mark: | 25f06d57 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6836/4/) | 2025-08-14 11:34:51 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-113056-25f06d-pr6836.zip) |
| :heavy_check_mark: | 25f06d57 | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/ios/job/PR-6836/4/) | 2025-08-14 11:36:16 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250814-113051-25f06d-pr6836.zip) |
| :x: | 25f06d57 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/5/) | 2025-08-14 11:38:28 | ~7 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/5/consoleText) |
| :heavy_check_mark: | 25f06d57 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/4/) | 2025-08-14 11:39:32 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/4//consoleText) |
| :heavy_check_mark: | 25f06d57 | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6836/4/) | 2025-08-14 11:42:40 | ~8 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250814-113355-25f06d-pr6836.zip) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | bb8101d6 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/android/job/PR-6836/5/) | 2025-08-14 11:45:54 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250814-114257-bb8101-pr6836.aar) |
| :heavy_check_mark: | bb8101d6 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6836/5/) | 2025-08-14 11:46:03 | ~3 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250814-114259-bb8101-pr6836.zip) |
| :heavy_check_mark: | bb8101d6 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6836/5/) | 2025-08-14 11:46:10 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-114304-bb8101-pr6836.zip) |
| :heavy_check_mark: | bb8101d6 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6836/5/) | 2025-08-14 11:47:18 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250814-114307-bb8101-pr6836.zip) |
| :heavy_check_mark: | bb8101d6 | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/ios/job/PR-6836/5/) | 2025-08-14 11:48:24 | ~5 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250814-114303-bb8101-pr6836.zip) |
| :heavy_multiplication_x: | bb8101d6 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/3/) | 2025-08-14 11:49:58 | ~6 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/3//consoleText) |
| :heavy_check_mark: | bb8101d6 | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6836/5/) | 2025-08-14 11:50:32 | ~7 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250814-114312-bb8101-pr6836.zip) |
| :x: | bb8101d6 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/6/) | 2025-08-14 11:53:31 | ~10 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/6/consoleText) |
| :heavy_check_mark: | bb8101d6 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/5/) | 2025-08-14 11:53:41 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/5//consoleText) |
| :heavy_check_mark: | bb8101d6 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/4/) | 2025-08-14 12:20:20 | ~24 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/4//consoleText) |
| | | | | | | |
| :x: | e234387d | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6836/6/) | 2025-08-14 12:06:18 | ~1 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6836/6/consoleText) |
| :x: | e234387d | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/7/) | 2025-08-14 12:07:05 | ~2 min | `linux` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6836/7/consoleText) |
| :x: | e234387d | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6836/6/) | 2025-08-14 12:07:15 | ~2 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6836/6/consoleText) |
| :x: | e234387d | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/android/job/PR-6836/6/) | 2025-08-14 12:07:29 | ~2 min | `android` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/android/job/PR-6836/6/consoleText) |
| :x: | e234387d | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6836/6/) | 2025-08-14 12:08:46 | ~3 min | `macos` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6836/6/consoleText) |
| :x: | e234387d | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6836/6/) | 2025-08-14 12:09:33 | ~4 min | `windows` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6836/6/consoleText) |
| :x: | e234387d | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/ios/job/PR-6836/6/) | 2025-08-14 12:09:58 | ~5 min | `ios` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/ios/job/PR-6836/6/consoleText) |
| :heavy_multiplication_x: | e234387d | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/6/) | 2025-08-14 12:11:37 | ~6 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6836/6//consoleText) |
| :heavy_multiplication_x: | e234387d | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/5/) | 2025-08-14 12:22:05 | ~1 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6836/5//consoleText) |

jakubgs committed:
fix(wakuv2)_: change NTPTimeSource to status-go TimeSource

Fix for errors like this:

messaging/core.go:421:3: cannot use params.timeSource
(variable of type "github.com/status-im/status-go/internal/timesource".TimeSource)
as *"github.com/status-im/status-go/timesource".NTPTimeSource value in argument to wakuv2.New: need type assertion

Introduced in:
https://github.com/status-im/status-go/pull/6829

Signed-off-by: Jakub Sokołowski <jakub@status.im>

jakubgs commented:
Superseded by:
- https://github.com/status-im/status-go/pull/6837
